### ISSUE TITLE
Do Not Disturb mode

### ISSUE THREAD

**Issue by iovis** [GoldenChaos/GoldenChaos-BTT#8]
Every time I screen share or want to have some heads down time I enable the system "Do Not Disturb" mode (in the Notifications Center).

Would love if this would be a toggle in the Menu Bar like Caffeinate!

**Comment by GoldenChaos** [comment]
Already got you covered! There's a DND switch in the control menu, it's the moon right next to the dark mode button 😄 

<img width="1085" alt="Touch Bar Shot 2019-09-16 at 7 54 53 AM" src="https://user-images.githubusercontent.com/1396727/64955874-50270680-d857-11e9-894a-d462cfddde18.png">

**Comment by iovis** [comment]
I suck, thanks for the patience and the good work!

**Comment by GoldenChaos** [comment]
In the future I'll hopefully find an easy way to let folks rearrange items in the control menu or pin them to the menu bar. But that might take a while, haha. 😄

**Comment by iovis** [comment]
That sounds like quite an undertaking, but I agree such a complex BTT config can be overwhelming for regular users. It took me a while to figure out why the icon was not showing in the menu bar when I copied it there till I found out the "show on key trigger" thingy buried in the options.

I'm sure there's people like me that use BTT only to run your config, so they won't be BTT ninjas and probably don't want to be, so I think it's a good idea.
